platform/ios/src/MGLMapboxEvents.m

```diff
@@ -8,6 +8,8 @@
 static NSString * const MGLMapboxMetricsDebugLoggingEnabled = @"MGLMapboxMetricsDebugLoggingEnabled";
 static NSString * const MGLTelemetryAccessToken = @"MGLTelemetryAccessToken";
 static NSString * const MGLTelemetryBaseURL = @"MGLTelemetryBaseURL";
+static NSString * const kMMEEventsProfile = @"MMEEventsProfile";
+static NSString * const kConfigHibernationRadiusVariableKey = @"VariableGeofence";
```

Comment:
The naming of `kConfigHibernationRadiusVariableKey` is a little unclear to me — this is the _value_ that the `kMMEEventsProfile` _key_ is checked for, so this shouldn’t include `Key` in its name.